subsys/bluetooth/mesh/Kconfig

```diff
@@ -51,6 +51,68 @@ endif # BT_MESH_RELAY
 
 endif
 
+config BT_MESH_NLC_PERF_SUPPORT
```

Comment:
A bit uncertain now after looking the whole code. May be instead of `_SUPPORT` it should be `_CONF` (configuration). If you disagree, then leave it as is.